README.md

```diff
@@ -123,6 +125,39 @@ net.train([{input: { r: 0.03, g: 0.7 }, output: { black: 1 }},
 
 var output = net.run({ r: 1, g: 0.4, b: 0 });  // { white: 0.81, black: 0.18 }
 ```
+#### For training with `RNN`, `LSTM` and `GRU`
+Each training pattern can either:
+* Be an array of values
+* Be a string
+* Have an `input` and an `output`
+  * Either of which can an array of values or a string
+
+CAUTION: When using an array of values, you can use ANY value, however, the values are represented in the neural network by a single input.  So the more _distinct values_ has _the larger your input layer_.  If you have a hundreds, thousands, or millions of floating point values _THIS IS NOT THE RIGHT CLASS FOR THE JOB_.  Also, when deviating from strings, this gets into beta
+
+Example using direct strings:
+```javascript
+var net = new brain.NeuralNetwork();
```

Comment:
Is this the correct way to initialize an `RNN`, `LSTM` or a `GRU`?  It seems like `new brain.NeuralNetwork();` is a copy paste mistake.